content/configuration/index.md

```diff
@@ -414,7 +415,16 @@ module.exports = {
   },
 
   [node](node): {
-    /* TODO */
+    // Polyfills and mocks to run Node.js-
+    // environment code in non-Node environments.
+    
+    [console](node#node-console): false, // boolean | "mock"
+    [global](node#node-global): true, // boolean | "mock"
+    [process](node#node-process): true, // boolean
+    [__filename](node#node-__filename): "mock", // boolean | "mock"
+    [__dirname](node#node-__dirname): "mock", // boolean | "mock"
+    [Buffer](node#node-buffer): true, // boolean | "mock"
+    [setImmediate](node#node-): true // boolean | "mock" | "empty"
```

Comment:
Thanks so much @skipjack -- you were absolutely right!

If I'm trying to submit a PR for this in the future, is there a reference for instructions on how I can see the compiled docs to avoid taking up time for typos like this in the future? :)